recipes/b2/portable/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile, tools
+import os
+
+
+class B2Conan(ConanFile):
+    name = "b2"
+    homepage = "https://boostorg.github.io/build/"
+    description = "B2 makes it easy to build C++ projects, everywhere."
+    topics = ("conan", "installer", "boost", "builder")
+    license = "BSL-1.0"
+    settings = "os_build", "arch_build"
```

Comment:
```suggestion
    settings = "os", "arch"
```

Keep os, arch only. os_build and arch_build will be deprecated for Conan 2.0: https://docs.conan.io/en/latest/devtools/build_requires.html#build-and-host-contexts